bokeh/_templates/autoload_js.js

```diff
@@ -76,7 +83,7 @@ calls it with the rendered model.
     console.log("Bokeh: BokehJS loaded, going straight to plotting");
```

Comment:
I did think about checking if `window._bokeh_is_loading == 0` that would at least prevent the race where a second script starts loading before the first embed has really finished. But won't help in the case where the first script loads without widgets but a second one does need widgets. I will make the change, though.